about.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Document</title>
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://use.typekit.net/gmf5adp.css">
+        <script src="https://kit.fontawesome.com/7bfd279a0b.js" crossorigin="anonymous"></script>
+    </head>
+    <body class="fade">
+        <header>
+            <div class="nav-banner">
+                <a href="#" class="banLink"><img src="./assets/icons/facebook.png"></a>
+                <a href="#" class="banLink"><img src="./assets/icons/twitter.png"></a>
+                <a href="#" class="banLink">English</a>
+                <a href="#" class="banLink">My Page</a>
+                <a href="#" class="banLink">Logout</a>
+            </div>
+        
+            <nav>
+                <div class="navbar">
+                <button class="menu-btn"><img src="./assets/icons/mobile_menu 1.png" alt=""></button>
+                  <div class="navbar-logo">
+                    <a class="fade" href="./home.html"><img src="./assets/logo/svg/Color logo - no background.svg" alt="Logo"></a>
+                  </div>
+                  <ul class="menu-items">
+                    <li><a class="menuLineItems fade-link" href="./home.html">Home</a></li>
+                    <li><a class="menuLineItems" href="#">Program</a></li>
+                    <li><a class="menuLineItems" href="home.html#JoinNPConnect">Join</a></li>
+                    <li><a class="menuLineItems" href="#">Sponsor</a></li>
+                    <li><a class="menuLineItems" href="#">News</a></li>
+                    <li id="campaign"><a class="menuLineItems link-fade" href="./home.html">NPConnect</a></li>
+                  </ul>
+                </div>
+              </nav>
+
+
+          </header>
+
+      <section class="frame4">
+        <div class="heroContent">
+            <div class="aboutContentGroup">
+                <div class="theme">
+                    <h3 id="themeSub">"Hello! Adventurer."</h6>
+                    <h1 id="themeTitle">NPConnect - What's it all about?
+                    </h1>
+                </div>
+                <div class="aboutText">
+                    <h4>Welcome to D&D Hub, the ultimate online destination for Dungeons and Dragons enthusiasts! Our platform is dedicated to providing a seamless and immersive gaming experience for players and Dungeon Masters (DMs) alike. Whether you're a seasoned adventurer or a newcomer to the realms of tabletop role-playing, D&D Hub is here to support your journey.
+                    </h4>
+                </div>
+            </div>
+            <div class="frame1">
+                <h4>Please contact us per Email for any further questions about NpConnect</h3>
+                    <a href="#"><p>NPConnect@Connectinggamers.com</p></a>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] The top section of the About page should follow the design given as well, this includes all issues already raised such as colors, and layout of the elements.
- it has one heading
- a paragraph
- a line of text 
- bold text at the bottom


Your design
![image](https://github.com/George7h/Module-1-capstone-project/assets/81932478/71e3dc0e-888b-4e5c-b89a-d2eced1e9769)

Given design
![image](https://github.com/George7h/Module-1-capstone-project/assets/81932478/ec871deb-2785-468f-951e-99948f70b41f)
 
### Important (Due to time):red_circle: 
#### Please once again note that the changes have to follow the layout, colors and typography of the given template for both the desktop and mobile view of the top section, middle section, and the bottom section

Reply:
My design didn't do this at all as you are showing.